.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
 .env
+/volume
```

Comment:
Interested to know what is in this /volume directory? doesn't exist on my machine